api/src/main/scala/dx/api/DxApi.scala

```diff
@@ -954,6 +954,28 @@ case class DxApi(version: String = "1.0.0", dxEnv: DXEnvironment = DxApi.default
     }
   }
 
+  // Copy object to destination project, if it isn't already there.
+  def cloneObject(id: String,
+                  sourceProject: DxProject,
+                  destProject: DxProject,
+                  destFolder: String = "/"): Unit = {
+    if (sourceProject.id == destProject.id) {
+      logger.trace(
+          s"""The source and destination projects are the same (${sourceProject.id}), 
+             |no need to clone object ${id}""".stripMargin.replaceAll("\n", " ")
+      )
+    } else {
+      logger.trace(s"Cloning object ${id} from ${sourceProject.id} to ${destProject.id}")
+      val request = Map("objects" -> JsArray(JsString(id)),
+                        "project" -> JsString(destProject.id),
+                        "destination" -> JsString(destFolder))
+      projectClone(sourceProject.id, request)
+
+      // TODO: Is this all that is necessary for cloning only file, not record?
+      // TODO: Could it work this way for both files and applets?
```

Comment:
Also, I looked at cloneAsset as an example, but want to exclude the parts that are only needed when cloning a record -- is it correct? @jdidion